src/tests/java/picard/analysis/TheoreticalSensitivityTest.java

```diff
@@ -202,16 +202,16 @@ public void testHetSensDistributions() throws Exception {
         final File wgsSampledMetricsFile = new File(TEST_DIR, "test_Solexa-316269_sampled.wgs_metrics");
 
         return new Object[][] {
-                {.9130, wgsMetricsFile},
-                {.9784, hsMetricsFile},
-                {.9562, targetedMetricsFile},
-                {.9892, wgsSampledMetricsFile}
+                {0.897_342_54, wgsMetricsFile},
```

Comment:
do you want a comment in the code? I'm treating these as regression tests. the original "approximate" values were calculated independently in the original R implementation.